static/app/views/feedback/feedbackListPage.tsx

```diff
@@ -88,7 +88,9 @@ export default function FeedbackListPage({}: Props) {
                     <Container style={{gridArea: 'list'}}>
                       <FeedbackList />
                     </Container>
-                    <FeedbackSearch />
+                    <SearchContainer>
+                      <FeedbackSearch />
+                    </SearchContainer>
```

Comment:
Since we're not using this, is there any shot we can put the style inline like line 94 below?

Reply:
i think we prefer non-inline styles? what do you mean by "not using this"

Reply:
Sorry, I meant "not reusing this"